judging/scripts/bootstrap.sh

```diff
@@ -11,6 +15,9 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+# What mode is this sandbox running in? "prescribe" or "predict"
+mode=$1
```

Comment:
The existing Phase 1 sandboxes will keep their (old) `run.sh` and `bootstrap.sh` files. These are the only two files that need to be deployed manually.